app/views/candidate_interface/restructured_work_history/job/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :title, title_with_error_prefix(t('page_titles.add_job'), @job_form.errors.any?) %>
+<% if current_application.application_work_experiences.present? %>
+  <% content_for :before_content, govuk_back_link_to(candidate_interface_work_history_show_path) %>
```

Comment:
Just noticed on my local env - this is taking me back to the old work history review